scripts/projectController.js

```diff
@@ -1,12 +1,24 @@
 (function(module) {
   var projectController = {};
 
-  Project.fetchAll();
-
-  projectController.index = function() {
+  projectController.index = function(ctx, next) {
+    console.log(ctx.projects);
+    console.log('we are taking the home route');
+    projectView.index(ctx.projects);
+  };
 
-    $('main > section').hide();
-    $('#projects').fadeIn();
+  projectController.loadAll = function(ctx, next) {
+    var projectData = function(allProjects) {
+      ctx.projects = Project.all;
+      next();
+    };
+    console.log(ctx);
+    if (Project.all.length) {
```

Comment:
This bit is causing the other part of your odd behavior. Consider what happens on first load vs later. What's the difference between them? Tracing the program execution path may help you here.